.github/workflows/tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  scaffold:
+    name: Scaffold Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "lts/*"
+      - name: Install dependencies
+        run: npm ci
+      - name: Run Era Test Node
+        uses: dutterbutter/era-test-node-action@latest
+      - name: hh-sol-hw scaffold
+        run: npm run dev -- create-project hh-sol-hw --framework hardhat_solidity --project hello_world && cd hh-sol-hw && yarn && yarn test
+      - name: hh-sol-ft scaffold
+        run: npm run dev -- create-project hh-sol-ft --framework hardhat_solidity --project fungible_token && cd hh-sol-ft && yarn && yarn test
+      - name: hh-sol-nft scaffold
+        run: npm run dev -- create-project hh-sol-nft --framework hardhat_solidity --project non_fungible_token && cd hh-sol-nft && yarn && yarn test
+      - name: hh-vyper-hw scaffold
+        run: npm run dev -- create-project hh-vyper-hw --framework hardhat_vyper --project hello_world && cd hh-vyper-hw && yarn && yarn test
+      - name: hh-vyper-ft scaffold
+        run: npm run dev -- create-project hh-vyper-ft --framework hardhat_vyper --project fungible_token && cd hh-vyper-ft && yarn && yarn test
+      - name: hh-vyper-nft scaffold
+        run: npm run dev -- create-project hh-vyper-nft --framework hardhat_vyper --project non_fungible_token && cd hh-vyper-nft && yarn && yarn test
```

Comment:
The actual test being run should take place in the CI of the template repo. If these tests fail, you won't fix them in the `zksync-cli` repo rather the template repo. What should be run here is unit tests for the `create-project` command.

Reply:
Valid. All of those upstream repos have a CI in them that perform those tests as well. As for the unit tests here I'd rather wait until #55 is merged and I'll implement the tests at that point